images/ceph/Dockerfile

```diff
@@ -23,8 +23,8 @@ ARG ARCH
 ARG TINI_VERSION
 
 # Run tini as PID 1 and avoid signal handling issues
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} /tini
-RUN chmod +x /tini
+RUN curl -SL -o /tini https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} && \
```

Comment:
I just ran the curl command to test it and we should supress the curl progress output and add the `--fail` flag.
`curl -sSL --fail -o /tini URL`